<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WombatRegistry</class>
 <widget class="QMainWindow" name="WombatRegistry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wombat Registry</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../wombatregistry/wombatregistry.qrc">
    <normaloff>:/wombat</normaloff>:/wombat</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treewidget">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="tablewidget">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
        <column/>
       </widget>
       <widget class="QPlainTextEdit" name="plaintext">
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenHive"/>
   <addaction name="actionManageTags"/>
   <addaction name="actionPreviewReport"/>
   <addaction name="actionPublish"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpenHive">
   <property name="icon">
    <iconset resource="../wombatregistry/wombatregistry.qrc">
     <normaloff>:/bar/openfolder</normaloff>:/bar/openfolder</iconset>
   </property>
   <property name="text">
    <string>Open Hive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionManageTags">
   <property name="icon">
    <iconset resource="../wombatregistry/wombatregistry.qrc">
     <normaloff>:/bar/managetags</normaloff>:/bar/managetags</iconset>
   </property>
   <property name="text">
    <string>Manage Tags</string>
   </property>
   <property name="toolTip">
    <string>Manage Tags</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../wombatregistry/wombatregistry.qrc">
     <normaloff>:/help</normaloff>:/help</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionPreviewReport">
   <property name="icon">
    <iconset resource="../wombatregistry/wombatregistry.qrc">
     <normaloff>:/bar/preview</normaloff>:/bar/preview</iconset>
   </property>
   <property name="text">
    <string>Preview Report</string>
   </property>
   <property name="toolTip">
    <string>Preview Report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPublish">
   <property name="icon">
    <iconset resource="../wombatregistry/wombatregistry.qrc">
     <normaloff>:/bar/publish</normaloff>:/bar/publish</iconset>
   </property>
   <property name="text">
    <string>Publish Report</string>
   </property>
   <property name="toolTip">
    <string>Publish Report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../wombatregistry/wombatregistry.qrc"/>
 </resources>
 <connections/>
</ui>
